tslint.json

```diff
@@ -19,6 +19,7 @@
 		"no-unnecessary-qualifier": true,
 		"no-unused-variable": true,
 		"no-var-keyword": true,
+		"object-literal-sort-keys": false,
```

Comment:
why was this added?

Reply:
tslint complains about objects not being alphabetically sorted otherwise, which makes the matrices for error testing a bit more unclear.